tests/Feature/SnmpTraps/CommonTrapTest.php

```diff
@@ -70,11 +70,11 @@ public function testFindByIp(): void
         $this->assertEquals($device->hostname, $trap->getDevice()->hostname);
 
         // check that eventlog was logged
-//        $eventlog = Eventlog::latest('datetime')->first();
-//        $this->assertEquals($device->device_id, $eventlog->device_id, 'Trap eventlog device incorrect');
-//        $this->assertEquals('', $eventlog->message, 'Trap eventlog message incorrect');
-//        $this->assertEquals('trap', $eventlog->type, 'Trap eventlog type incorrect');
-//        $this->assertEquals(2, $eventlog->severity, 'Trap eventlog severity incorrect');
```

Comment:
Oh it sorts dates different :)

Reply:
I guess, just sorted by index.  That should always be in the correct order.